tests/urls.py

```diff
@@ -1,30 +1,38 @@
 from django.urls import include, path
-
 from rest_framework import routers
 
 from resolwe.api_urls import api_router as resolwe_router
 from resolwe.flow.views import EntityViewSet
+
 from resolwe_bio.filters import BioEntityFilter
-from resolwe_bio.kb.views import (
-    FeatureViewSet, MappingSearchViewSet
-)
+from resolwe_bio.kb.views import FeatureViewSet, MappingSearchViewSet
+from resolwe_bio.variants.views import VariantAnnotationViewSet, VariantViewSet
 
 from .routers import SearchRouter
 
-
 EntityViewSet.filterset_class = BioEntityFilter
 
 api_router = routers.DefaultRouter(trailing_slash=False)
-api_router.register(r'sample', EntityViewSet)
+api_router.register(r"sample", EntityViewSet)
```

Comment:
Can you move reformatting to a separate commit and keep only relevant changes here?

Reply:
Sure.